public/css/style.css

```diff
@@ -357,6 +357,10 @@ label {
   margin: 0;
 }
 
+#notification-icon {
```

Comment:
To avoid this, just make it a link versus the i tag. Include the i tag as the link HTML, though. More proper HTML and removes the need for setting a cursor type.